pwa/src/templates/templateParts/userForm/UserFormTemplate.tsx

```diff
@@ -79,6 +85,13 @@ export const UserFormTemplate: React.FC<UserFormTemplateProps> = ({ user }) => {
     );
   }, [getOrganization.isSuccess]);
 
+  React.useEffect(() => {
+    setValue(
+      "applications",
+      user["applications"].map((application: any) => ({ label: application.name, value: application.id })),
+    );
+  }, [getOrganization.isSuccess]);
```

Comment:
As you've mentioned on Slack, this dependency array is incorrect. However, I don't think the solution is rewriting it to `getApplication.isSuccess`, because we're not actually targeting the `getApplication.data` to set the value.

A better solution, I think, would be to add this `setValue(...)` to the `handleSetFormValues`, like so:


```
const handleSetFormValues = (user: any): void => {
  const basicFields: string[] = ["name", "description", "email", "password", "locale", "person"];
  basicFields.forEach((field) => setValue(field, user[field]));

  setValue(
    "applications",
    user.applications.map((application: any) => ({ label: application.name, value: application.id })), // <= NOTE: user always exists here, because we only run this function if the user is set :)
  );
};
```